Gemfile

```diff
@@ -54,7 +54,6 @@ gem 'pry-rails'
 group :development do
   gem 'pry-remote'
   gem 'pry-doc'
-  gem 'debug'
```

Comment:
For some reason, `debug`'s dependencies were not being added to the lock file. They were earlier. No idea why. For now, I'm just removing it.